aiida/tools/importexport/archive/writers.py

```diff
@@ -328,9 +322,13 @@ def write_entity_data(self, name: str, pk: int, id_key: str, fields: Dict[str, A
             self._data['node_extras'][pk] = fields.pop('extras')
         self._data['export_data'].setdefault(name, {})[pk] = fields
 
-    def write_node_repo_folder(self, uuid: str, path: Union[str, Path], overwrite: bool = True):
+    def write_repository_container(self, container: Container):
+        """Write a repository container to the archive.
+
+        :param container: the container.
+        """
         self.assert_within_context()
-        (self._archivepath / NODES_EXPORT_SUBFOLDER / export_shard_uuid(uuid)).puttree(path, check_exists=not overwrite)
+        (self._archivepath / 'container').puttree(container.get_folder())
```

Comment:
this is currently "double" compressing the object-store

Reply:
I can set `compress=False` in the exporting of the container itself